users/models.py

```diff
@@ -7,5 +7,20 @@
 
 class Profile(models.Model):
     user = models.OneToOneField(
-        User, on_delete=models.CASCADE, primary_key=True)
-    nick_name = models.CharField('Nick name', max_length=30, null=True)
+        User, related_name='user', on_delete=models.CASCADE, primary_key=True)
+    nick_name = models.CharField(
+        'Nick name', max_length=30, blank=True, default='')
+
+    def __str__(self):
+        return self.user.username
+
+
+def create_profile(sender, **kwargs):
+    user = kwargs["instance"]
+    if kwargs["created"]:
+        profile = Profile(user=user)
+        profile.nick_name = ''
+        profile.save()
+
+
+post_save.connect(create_profile, sender=User)
```

Comment:
I don't think we need this. I had done fixes that already showed the profile fields under the user in the admin section.

Reply:
In the repo's version, when you create a superuser it does not create a profile( One ---> Zero or One). With these changes, there is always a profile for every user( One ---> One)